tests/page-objects/collection-page.ts

```diff
@@ -43,85 +64,12 @@ export class CollectionPage {
   }
 
   async clickMoreBtnFromSummary() {
-    await expect(this.page.locator('#page-header')).toBeVisible();
+    await this.pageHeader.waitFor({ state: 'visible' });
     await this.pageSummary.getByTestId('collection-page-more-link-btn').click();
   }
 
-  async validatePageHeaderElements() {
-    await expect(this.page.locator('#page-header')).toBeVisible();
-    await expect(
-      this.page.locator('#top-matter > div.thumbnail-frame'),
-    ).toBeVisible();
-    await expect(this.pageSummary).toBeVisible();
-    await expect(this.page.locator('action-bar')).toBeVisible();
-  }
-
-  async validateCollectionPageTabs() {
-    await expect(this.pageTabs).toBeVisible();
-    // this could cause an error in some detailsPage that doesn't have Forum tab like ytjdradio
-    // should be tackled in a different task
-    await expect(
-      this.pageTabs.getByLabel('Collection', { exact: true }),
-    ).toBeVisible();
-    await expect(this.pageTabs.getByLabel('Forum')).toBeVisible();
-    await expect(this.pageTabs.getByLabel('About')).toBeVisible();
+  async getPageActiveTabText() {
+    return await this.pageTabs.locator('li.tab.active').innerText();
```

Comment:
```suggestion
    return this.pageTabs.locator('li.tab.active').innerText();
```

(since we're returning a promise anyway)

Reply:
<img width="465" alt="Screenshot 2025-05-26 at 10 59 26" src="https://github.com/user-attachments/assets/20e8837f-dfcf-4410-9e1f-5cad35d77409" />

`.innerText()` returns `Promise<string>`

Reply:
That makes sense. My point is just that `return await` appears to be redundant here, since the return value of `getPageActiveTabText` is also a `Promise<string>` by virtue of it being `async`. We can just return the original promise itself if we don't care about the actual resolved/rejected result here.